src/bin/solang.rs

```diff
@@ -64,9 +64,15 @@ fn main() {
             Arg::new("OPT")
                 .help("Set llvm optimizer level")
                 .short('O')
+                .use_value_delimiter(false)
+                .long_help("\
+None = 0,      // -O0
+Less = 1,      // -O1
+Default = 2,   // -O2, -Os, -OS
+Aggressive = 3 // -O3")
```

Comment:
`cargo fmt` does not like it and I'm not a fan either. No need to mention none/less/default/aggressive.